Automattic/woocommerce-payments
# Redirect "ch_" charge id format to "pi_" payment intent id on Transaction Detail screens

Fixes #3122

#### Changes proposed in this Pull Request



Redirect "ch_" charge id format to "pi_" payment intent id on Transaction Detail screens



Update the Transaction Details screen to allow receiving a "pi_" Payment Intent ID instead of a "ch_" Charge ID. This change should create more consistency between the links and the URLs on the client.





#### Testing instructions





* Perform an order on your test shop
* Go to **WooCommerce -> Orders**
* Open on the order that you have made
* Click on the Payment Intent id link 
![image](https://user-images.githubusercontent.com/16882226/173832383-39fedf12-39e6-451f-9990-33467c1de14e.png)
* It should have "ch_" on the query param and then redirect to the "pi_" equivalent

-------------------


**Post merge**
